parasl/parasl.txt

```diff
@@ -68,7 +81,10 @@ arr_inputs = input(0..3) : int[4]; // array of 4
 
 1.4. Structures
 
-Structure entity is a syntactic glue for the couple of entities of different types.
+Structure entity is a syntactic glue for the couple of entities of different
+types. You can define structure using glue expression that takes expressions as
```

Comment:
Strictly saying you are not "defining structure". Youre doing something like defining structure in C implicitly. But actually you are gluing objects.